### ISSUE TITLE
Detection with the original YOLO3

### ISSUE THREAD

**Issue by blingojames** [NeuralSec/Daedalus-attack#2]
Hi, thank you very much for the project,
would appreciate your help,
We're trying to run the adversarial examples from your site, as they are
over the original Yolo3 (the C version) and it does detect a person,
For example we tried taking 'Best\ example\ of\ 5\ Distortion\ 99.68110656738281.png' from this site, and run it as it is over the Original Yolo3, and we do see a person detection in Yolo's output.
Are we doing something wrong? or maybe the examples need to be run against the detector they were trained upon?

Best regards,
Blingo

**Comment by markvex** [comment]
Hi,
i have similar issue- how can i convert the results from my training to an adversarial image?
thanks

**Comment by NeuralSec** [comment]
> Hi, thank you very much for the project,
> would appreciate your help,
> We're trying to run the adversarial examples from your site, as they are
> over the original Yolo3 (the C version) and it does detect a person,
> For example we tried taking 'Best\ example\ of\ 5\ Distortion\ 99.68110656738281.png' from this site, and run it as it is over the Original Yolo3, and we do see a person detection in Yolo's output.
> Are we doing something wrong? or maybe the examples need to be run against the detector they were trained upon?
> 
> Best regards,
> Blingo

Hi,

That's wired. You may try a few fixes as follows:
1. In case that yolov3 has updated its weights. You can download the weights [here](https://drive.google.com/open?id=1-1UTBXnqH0-nC0CVIi8T-8hSzoxENgNK).
2. Please make sure that the adversarial example has not gone through any forms of compression or randomization before detection.

Regards,

**Comment by LenaNans** [comment]
Hello, 
I am attempting to run the l2_yolov3 section of code and coming across the error:
"AbortedError (see above for traceback): Operation received an exception:Status: 3, message: could not create a dilated convolution forward descriptor, in file tensorflow/core/kernels/mkl_conv_ops.cc:1111".
I have traced this error to some of the loss function calculations in the "attack" method. It looks like the loss values are not being stored properly in a tensor. This issue might be tied with the newest version of TF& Keras, but any help regarding this issue would be greatly appreciated. 
Thank you for your time. 
-Lena
